ecs.php

```diff
@@ -24,6 +24,9 @@
         '*/Fixture/*',
         '*/Expected/*',
 
+        // avoid re-running on build
+        __DIR__ . '/preload.php',
```

Comment:
also `preload-split-package.php`:

```suggestion
        __DIR__ . '/preload.php',
        __DIR__ . '/preload-split-package.php',
```